src/lib.rs

```diff
@@ -375,6 +375,9 @@ pub mod doctest_tcell;
 #[cfg(feature = "std")]
 pub mod doctest_tlcell;
 
+#[cfg(feature = "generativity")]
+pub extern crate generativity;
```

Comment:
qcell is edition = "2018", is there a reason this extern crate definition is needed?

Reply:
oh it's `pub`, I guess that makes sense.